Cargo.toml

```diff
@@ -8,22 +8,22 @@ license = "MIT OR Apache-2.0"
 repository = "https://github.com/mir-protocol/r1cs"
 documentation = "https://docs.rs/r1cs"
 keywords = ["R1CS", "cryptography", "SNARK"]
-categories = ["cryptography"]
+categories = ["cryptography", "no-std"]
 maintenance = { status = "actively-developed" }
 edition = "2018"
 
 [features]
-default = ["bimap", "itertools", "num", "num-traits"]
-no-std = []
+default = ["std"]
```

Comment:
This should be set to ```default = []``` to make the default compile `no_std` and avoid requiring the `--no-default-features` flag.